### ISSUE TITLE
Validation Loss as well as validation F1 increasing

### ISSUE THREAD

**Issue by kuldeep7688** [allenai/scibert#69]
Hi, 
While finetuning scibert on JNLPBA and bc5cdr dataset, I observed very strange phenomenon.
The validation loss is increasing whereas the validation F1 in either increasing or staying the same. 

Please let me know because I have faced a similar issue when I used pytorch-crf over scibert. There also for ner (JNLPBA and bc5cdr) validation loss anf validation f1 are increasing.
